### ISSUE TITLE
Problem with questions that are stored in .json file

### ISSUE THREAD

**Issue by Andrei6700-exe** [Andrei6700/ChatBot#1]
When I ask questions that are not in the .json file and questions from math, I get this answer in the terminal at test-API.py:
"gradient descent algorithm is an optimized algorithm used to minimize the loss function in machine learning models it........... "
Possiblile fix: Revert the commit

**Comment by Andrei6700** [comment]
i resolve some problems about math problems !
